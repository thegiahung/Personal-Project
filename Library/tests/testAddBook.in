Add book books.csv 200000
Add book books.csv 111111
Add book books.csv 111112
Add book books.csv 111111
Add book hello.csv 111119
Exit
