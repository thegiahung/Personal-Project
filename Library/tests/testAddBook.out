user: No such book in file.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Successfully added: The Hobbit (J.R.R. Tolkien).

user: Book already exists in system.

user: No such file.

user: Ending Library process.
